### PR TITLE
adding s-band status to meerkat status script

### PR DESCRIPTION
**Pull request by Duocasius**
Added s-band sensor logic to meerkat status script.

### CODE REVIEW

**Review by TiyaniBaloyi — CHANGES_REQUESTED** [review]
The script works just fine. I have identified a few areas to refactor;
For consistency, let's rename:
- `last global sync` to `Last global sync`
- `time left until next sync` to `Time left until next sync`
- `current obs` to `Current obs`, and
- `available antennas` to `Available antennas`
      
    image

Also, let's rename the following table sensors:
- `e-Stop` to `e-stop`
- `Amp2` to `amp2`

File: utility/meerkat_status.py [hunk 1]
```diff
@@ -137,18 +137,14 @@ def get_rsc_sensors(ant, band):
         lna_h = rsc_values.pop("rsc_rx{}_mmic_enable.mmic1".format(band))
```
Comment (TiyaniBaloyi) [inline utility/meerkat_status.py:137]: you have repeated terms for `rsc_values["lnas"]`, `amp2_h`, 'amp2_v' and `rsc_values["amp2"]` on both conditions for setting `lna_h` and `lna_v`.

I would change this to:
```py
if band == "s":
    lna_h = rsc_values.pop("rsc_rx{}_mmic_enable.mmic1".format(band))
    lna_v = rsc_values.pop("rsc_rx{}_mmic_enable.mmic2".format(band))
else:
    lna_h = rsc_values.pop("rsc_rx{}_lna_h_power_enabled".format(band))
    lna_v = rsc_values.pop("rsc_rx{}_lna_v_power_enabled".format(band))
 
rsc_values["lnas"] = "ON" if lna_h and lna_v else "OFF"
amp2_h = rsc_values.pop("rsc_rx{}_amp2_h_power_enabled".format(band))
amp2_v = rsc_values.pop("rsc_rx{}_amp2_v_power_enabled".format(band))
rsc_values["amp2"] = "ON" if amp2_h and amp2_v else "OFF"   
```

**Review by TiyaniBaloyi — CHANGES_REQUESTED** [review]
Code refactoring.

File: utility/meerkat_status.py (hunk 1 above)
Comment (Duocasius) [reply utility/meerkat_status.py:137]: We don't have second stage amplifier sensors connected to the S-band packetizers(or at least it's not listed under the same sensor name). Had to trick the script to think that we do have.

**Review by TiyaniBaloyi — APPROVED** [review]

**Review by bngcebetsha — APPROVED** [review]
@Duocasius I'm going to assume the script still works :), changes look good to  me

**Comment by Duocasius** [comment]
@bngcebetsha yes, it still works.

**Comment by lanceWilliams** [comment]
I'm not familiar with this script or the particular changes required for s band but since it is based on a resource container I don't foresee any issues with these changes.

**Review by lanceWilliams — APPROVED** [review]
Looks ok.

**Pull request merged**